downstream/modules/platform/proc-aap-migration.adoc

```diff
@@ -0,0 +1,7 @@
+[id="aap-migration_{context}"]
+
+= Migrating data to the {PlatformNameShort} Operator
+
+[role=_abstract]
+
```

Comment:
Is there just one step to this procedure? It almost doesn't read like a procedure. Seems more conceptual with the way it's written. See suggestion below.